elide-integration-tests/src/test/java/com/yahoo/elide/auditTests/AuditIT.java

```diff
@@ -127,4 +129,25 @@ public void testAuditUpdateOnInverseCollection() {
         Assert.assertTrue(logger.logMessages.contains("Entity with id 1 now has inverse list []"));
         Assert.assertTrue(logger.logMessages.contains("Inverse entities: []"));
     }
+
+    @Test(priority = 999)
+    public void testAuditWithDuplicateLineageEntry2() {
```

Comment:
This test will need to go in a new class that extends `AbstractIntegrationTestInitializer` and creates a `StandardTestBinder` that returns the appropriate status code.